packages/frontend/src/lib/table/model/ModelColumnActions.svelte

```diff
@@ -5,8 +5,13 @@ import { faFolderOpen } from '@fortawesome/free-solid-svg-icons';
 import ListItemButtonIcon from '../../button/ListItemButtonIcon.svelte';
 import { studioClient } from '/@/utils/client';
 import { router } from 'tinro';
+import { onMount } from 'svelte';
+import { inferenceServers } from '/@/stores/inferenceServers';
 export let object: ModelInfo;
 
+let inUse: boolean = false;
+$: inUse;
```

Comment:
What is the use of this declaration?

Reply:
https://github.com/containers/podman-desktop-extension-ai-lab/blob/5cfee5855e70a1c7f82fb9062b6d5f9ebdc971bc/packages/frontend/src/lib/table/model/ModelColumnActions.svelte#L42

It is updated by a store subscription, therefore it need to be marked as reactive, otherwise it does not get updated

Reply:
By removing it (and the `$: status;` in `ModelColumnIcon.svelte`), they are still updated. Perhaps a change on Svelte 5?
 
https://github.com/containers/podman-desktop-extension-ai-lab/assets/9973512/882cd191-df12-459e-94a8-b0a32e0e7003

Reply:
That would be a good news actually !

Reply:
Unit tests are failing when removing it

Reply:
ok, let's keep it as this then